packages/website/docs/creating-multi-source-autocompletes.md

```diff
@@ -2,3 +2,375 @@
+
+## Prerequisites
+
+This tutorial assumes that you have:
+- a populated [Query Suggestions Algolia index](https://www.algolia.com/doc/guides/building-search-ui/ui-and-ux-patterns/query-suggestions/how-to/creating-a-query-suggestions-index/js/)
+- existing markup containing an input element where you want to implement the autocomplete dropdown
+- front-end development proficiency with HTML, CSS, and JavaScript
+
+:::note
+
+If you don't have a Query Suggestions index yet, follow the guide on [creating a Query Suggestions index](https://www.algolia.com/doc/guides/building-search-ui/ui-and-ux-patterns/query-suggestions/how-to/creating-a-query-suggestions-index/js/).
+
+:::
+
+To follow along with the tutorial, you can start with the markup and style sheets from the [complete example on GitHub](https://github.com/algolia/doc-code-samples/tree/autocomplete-v1/Autocomplete/multi-source).
+
+## Getting started
+
+First, begin with some boilerplate for the autocomplete implementation—create a file called `index.js` in your `src` directory, and add the boilerplate below:
+
+```js
+import { autocomplete } from '@algolia/autocomplete-js';
+
+// Instantiate the autocomplete instance
+autocomplete({
+  container: '#autocomplete',
+  plugins: [],
+  openOnFocus: true,
+});
+```
+
+This boilerplate assumes you want to insert the autocomplete into a DOM element with `autocomplete` as an [id](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id). You should change the [`container`](autocomplete-js/#container) to [match your markup](basic-options). Setting [`openOnFocus`](autocomplete-js/#openonfocus) to `true` ensures that the dropdown appears as soon as a user clicks on the input.
+
+For now, `plugins` is an empty array, but you'll learn how to create and add plugins for predefined items, recent searches, and Query Suggestions next.
+
+## Adding predefined items
+
+A popular search pattern for autocomplete menus shows predefined search terms as soon as a user clicks on the search bar and before they begin typing anything. This UX provides a guided experience and exposes users to helpful resources or other content you want them to see.
+
+This tutorial describes how to create a plugin to show static, predefined items. In particular, it exposes helpful links the user may want to refer to.
+
+### Creating a predefined items plugin
+
+Begin by creating a `predefinedItemsPlugin.js` file in your `src` directory, with the following code:
+
+```js
+const predefinedItems = [
+  {
+    label: 'Documentation',
+    url: 'https://autocomplete.algolia.com/',
+  },
+  {
+    label: 'GitHub',
+    url: 'https://github.com/algolia/autocomplete.js/tree/next',
+  },
+]
+
+export const predefinedItemsPlugin = {
+  getSources() {
+    return [
+      {
+        getItems({ query }) {
+          return predefinedItems.filter(
+            (item) =>
+              !query || item.label.toLowerCase().includes(query.toLowerCase())
+          );
+        },
```

Comment:
This might be easier to understand and to modify for users, wdyt?

```suggestion
        getItems({ query }) {
          if (!query) {
            return predefinedItems;
          }
          
          return predefinedItems.filter(
            (item) =>
              item.label.toLowerCase().includes(query.toLowerCase())
          );
        },
```